gtd-timer/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -47,6 +50,7 @@ public Task HandleExceptionAsync(HttpContext context, Exception exception)
         private HttpStatusCode GetExceptionCode(Exception exception)
         {
             HttpStatusCode exceptionCode;
+            this.logger.LogError("Error message: {0}; Error occured in: {1}, {2}", exception.Message, exception.Source,exception.TargetSite);
```

Comment:
can you use c#'s string interpolation here? also please format this better

Reply:
Change (in that way?)
![chrome_2019-01-20_16-33-39](https://user-images.githubusercontent.com/22987347/51440569-456c9f80-1cd1-11e9-8dc0-43a766bba36e.png)